examples/catalog/bin/sample_page.dart

```diff
@@ -22,22 +22,29 @@ class SampleError extends Error {
 // beginning of lines within the comment. A keyword's value is all of
 // the following text up to the next keyword or the end of the comment,
 // sans leading and trailing whitespace.
-const String sampleCatalogKeywords = r'^Title:|^Summary:|^Description:|^Classes:|^Sample:|^See also:';
+const String sampleCatalogKeywords =
+    r'^Title:|^Summary:|^Description:|^Classes:|^Sample:|^See also:';
 
 Directory outputDirectory;
 Directory sampleDirectory;
 Directory testDirectory;
 Directory driverDirectory;
 
-void logMessage(String s) { print(s); }
-void logError(String s) { print(s); }
+void logMessage(String s) {
+  print(s);
+}
+
+void logError(String s) {
+  print(s);
+}
```

Comment:
i'm not clear that this is an improvement, actually. it's one of the least controversial changes, but even this one, if you look at the original code and compare it to the new code, i think it's clear that the original code is easier to understand. It should the parallels between these two functions much more obviously.

Reply:
Dartfmt and the analyzer are perfectly happy with
```dart
void logMessage(String s) => print(s);
void logError(String s) => print(s);
```
I think that is cleaner than both the original code and the dartfmt version.

Reply:
does this sounds reasonable? if so I can add it as an automatic quick fix.

Reply:
https://github.com/dart-lang/dart_style/issues/452 talks about doing this if there's an explicit `return`.

I don't think it makes sense to use `=>` when you're not returning something (as in the `void` case), because it's a bit misleading.

In this case the formatting is IMHO better as it used to be because these methods are almost identical. It draws attention to their similarity. If they were unrelated, or had different signatures, or different bodies, then IMHO it would be more legible as separate blocks.

I think a formatter could totally figure this out using heuristics. Not sure how far down that road you'd want to go. For example, maybe sequences of methods with the same signature and same body, all of which would fit on one line, should be collapsed in this way, but if any of the methods wouldn't fit on one line, then none of them should be done that way.

At the end of the day, this is definitely not the top priority. If it was the only issue left then I would treat it as nice-to-have and punt on it. If it was one of a dozen nice-to-have issues then we'd want to make sure most of them were resolved, in which case maybe this one would be better to solve than some other more thorny issue.

Reply:
Great. Glad this is something we can worry about in the future after more significant deltas are resolved.

Reply:
> I don't think it makes sense to use => when you're not returning something (as in the void case), because it's a bit misleading.

I agree, but that ship has sailed. Users use `=>` for setters all the time.